src/main/java/ca/corefacility/bioinformatics/irida/ria/web/services/UICartService.java

```diff
@@ -5,38 +5,44 @@
 
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.context.MessageSource;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
 import org.springframework.stereotype.Component;
 
 import ca.corefacility.bioinformatics.irida.model.project.Project;
 import ca.corefacility.bioinformatics.irida.model.sample.Sample;
 import ca.corefacility.bioinformatics.irida.ria.web.ajax.dto.CartSampleModel;
 import ca.corefacility.bioinformatics.irida.ria.web.ajax.dto.cart.CartProjectModel;
 import ca.corefacility.bioinformatics.irida.ria.web.cart.dto.AddToCartRequest;
+import ca.corefacility.bioinformatics.irida.ria.web.cart.dto.CartSamplesByUserPermissions;
 import ca.corefacility.bioinformatics.irida.ria.web.cart.dto.CartUpdateResponse;
 import ca.corefacility.bioinformatics.irida.ria.web.components.ant.notification.ErrorNotification;
 import ca.corefacility.bioinformatics.irida.ria.web.components.ant.notification.Notification;
 import ca.corefacility.bioinformatics.irida.ria.web.components.ant.notification.SuccessNotification;
 import ca.corefacility.bioinformatics.irida.ria.web.components.ant.notification.WarnNotification;
 import ca.corefacility.bioinformatics.irida.ria.web.sessionAttrs.Cart;
+import ca.corefacility.bioinformatics.irida.security.permissions.sample.UpdateSamplePermission;
 import ca.corefacility.bioinformatics.irida.service.ProjectService;
 import ca.corefacility.bioinformatics.irida.service.sample.SampleService;
 
 /**
- * Service for handling all aspects interaction with the Cart.
+ * Service for handling all aspects' interaction with the Cart.
```

Comment:
Remove the `'` and add `of` after the word aspects

Reply:
Removed in d29431c0f3a66716de6beb8315f1e1a95040c391

Reply:
The `'` was removed but you didn't add `of`. Should be `Service for handling all aspects of interaction with the Cart.`

Reply:
Crap.... sorry.  Fixed in b2cd6d3